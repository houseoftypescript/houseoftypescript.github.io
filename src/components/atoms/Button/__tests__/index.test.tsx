import { render } from '@testing-library/react';
import { Button } from '../index';

describe('Button', () => {
  it('renders correctly', () => {
    const button = render(<Button></Button>);
    expect(button.container).toMatchSnapshot();
  });
});
