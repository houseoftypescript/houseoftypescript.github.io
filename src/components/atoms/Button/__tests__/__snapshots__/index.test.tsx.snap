// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<div>
  <button
    class="px-8 py-4 rounded-md bg-emerald-500 hover:bg-gray-900 text-white transition duration-500 "
    type="button"
  />
</div>
`;
